rusk/src/lib/chain/rusk.rs

```diff
@@ -202,7 +203,7 @@ impl Rusk {
         block_gas_limit: u64,
         generator: &StakePublicKey,
         txs: &[Transaction],
-        missed_generators: &[StakePublicKey],
+        slashing: Vec<Slash>,
```

Comment:
For the record, it's weird to have a function called `verify_transactions` do more than that. I think we should rename it.

Reply:
Agree, to properly record it you should open an issue

Reply:
I'll add it to the tech debt list